Ricochet/Assets/_Scripts/UI/_UICharacterSelectView.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class _UICharacterSelectView : MonoBehaviour {
+	#region Inspector Variables
+	[Header("Player Sprites")]
+    [Tooltip("No selection sprite (should be enabled)")]
+    [SerializeField]
+    private GameObject noSelect;
+    [Tooltip("Red Bean Sprite")]
+	[SerializeField]
+	private GameObject redBean;
+	[Tooltip("Blue Bean Sprite")]
+	[SerializeField]
+	private GameObject blueBean;
+	[Tooltip("Red Carrot Sprite")]
+	[SerializeField]
+	private GameObject redCarrot;
+	[Tooltip("Blue Carrot Sprite")]
+	[SerializeField]
+	private GameObject blueCarrot;
+
+	[Header("Cursor Sprites")]
+	[Tooltip("Bean Cursor")]
+	[SerializeField]
+	private GameObject beanCursor;
+	[Tooltip("Carrot Cursor")]
+	[SerializeField]
+	private GameObject carrotCursor;
+    #endregion
+
+    #region Private Variables
+    private bool active = false;
+    private int currentTeam = 1;
+	private string currentSpriteString;
+    private GameObject currentSprite;
+    private GameObject currentCursor;
+    #endregion
+
+    #region MonoBenavior
+    private void Awake()
+    {
+		currentSpriteString = "bean";
+		currentSprite = noSelect;
+		currentCursor = beanCursor;
+    }
+    #endregion
+
+    #region UI Behaviour
+    public void UpdateCharacter(string newCharacter)
```

Comment:
Shouldn't be string comparisons. This should be an enum. Strings are too finicky and prone to errors

Reply:
Can we leave this for now but create a task for austen and pinak to deal with this since I don't really know how this all works.